test_scripts/API/DeleteCommand/ATF_DeleteCommand_VR_DeleteCommand_WARNINGS.lua

```diff
@@ -0,0 +1,191 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one HMI-portions
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this test case when VR.DeleteCommand gets WARNINGS is checked
+-- 1. Used preconditions: App is activated and registered SUCESSFULLY
```

Comment:
@vvvakulenko, will be better to print "registered and activated" instead of "activated and registered" .